brewtils/resolvers/parameter.py

```diff
@@ -124,6 +128,12 @@ def _recurse_and_resolve(self, parameter, *_):
                 except (IndexError, ValueError):
                     pass
 
+            elif BYTES_PREFIX in parameter:
+                try:
+                    parameter = self.simple_resolve(parameter)
+                except (IndexError, ValueError):
```

Comment:
It's weird to me that this doesn't throw an error

Reply:
Yeah, IDK. This has been rewritten.